<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestfour</class>
 <widget class="QMainWindow" name="gestfour">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>689</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabSponsor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1231</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Fournisseur</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>1151</width>
        <height>501</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 238);
background-color: rgb(233, 255, 254);</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QTableView" name="tab_fournisseur">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>110</y>
         <width>501</width>
         <height>261</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>461</width>
         <height>491</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string/>
       </property>
       <widget class="QLabel" name="label_ID">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ID _Fournissuer:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Nom">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>161</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nom_Fournisseur:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Adr">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Adresse Mail :</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_ID">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>39</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Nom">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>79</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 207);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_Adr">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>119</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">

border-color: rgb(0, 0, 255);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
        <property name="inputMethodHints">
         <set>Qt::ImhNone</set>
        </property>
        <property name="placeholderText">
         <string>exemple@gmail.com</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_ajouter">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>250</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">
color: rgb(35, 170, 102);
</string>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Adr_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>N° Téléphone : </string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_tlf">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>159</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_modifier">
        <property name="geometry">
         <rect>
          <x>230</x>
          <y>250</y>
          <width>161</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">
color: rgb(35, 170, 102);
</string>
        </property>
        <property name="text">
         <string>Modifier</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>520</x>
          <y>210</y>
          <width>181</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Séléctionnez une image</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_img_2">
        <property name="geometry">
         <rect>
          <x>540</x>
          <y>250</y>
          <width>161</width>
          <height>161</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">capacity:10%;</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_Adr_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Status</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>200</y>
          <width>82</width>
          <height>28</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>active</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>inactive</string>
         </property>
        </item>
       </widget>
       <widget class="QLineEdit" name="lineEdit_id_sup">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>360</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>360</y>
          <width>41</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>ID :</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pb_supp">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>350</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>OCR A Extended</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(35, 170, 102);</string>
        </property>
        <property name="text">
         <string>Supprimer</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Medicament">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>390</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Medicament</string>
        </property>
       </widget>
       <widget class="QPushButton" name="client">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>390</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Client</string>
        </property>
       </widget>
       <widget class="QPushButton" name="employer">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>390</y>
          <width>93</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>Employer</string>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="label_recherche">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>70</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ID :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_recherche">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>70</y>
         <width>211</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_recherche">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>60</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>OCR A Extended</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(216, 191, 0);
color: rgb(35, 170, 102);</string>
       </property>
       <property name="text">
        <string>Recherche</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_recherche_2">
       <property name="geometry">
        <rect>
         <x>1000</x>
         <y>110</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(35, 170, 102);</string>
       </property>
       <property name="text">
        <string>Tri :</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_id">
       <property name="geometry">
        <rect>
         <x>1000</x>
         <y>140</y>
         <width>97</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_nom">
       <property name="geometry">
        <rect>
         <x>1000</x>
         <y>170</y>
         <width>121</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nom société</string>
       </property>
      </widget>
      <widget class="QLabel" name="countdown">
       <property name="geometry">
        <rect>
         <x>1030</x>
         <y>0</y>
         <width>131</width>
         <height>51</height>
        </rect>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_PDF">
       <property name="geometry">
        <rect>
         <x>990</x>
         <y>200</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>OCR A Extended</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(35, 170, 102);</string>
       </property>
       <property name="text">
        <string>PDF</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pb_STAT">
       <property name="geometry">
        <rect>
         <x>990</x>
         <y>260</y>
         <width>131</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>OCR A Extended</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(35, 170, 102);</string>
       </property>
       <property name="text">
        <string>STATS</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Search_button">
       <property name="geometry">
        <rect>
         <x>810</x>
         <y>10</y>
         <width>161</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>OCR A Extended</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">
color: rgb(35, 170, 102);
</string>
       </property>
       <property name="text">
        <string>Web Search</string>
       </property>
      </widget>
      <widget class="QPushButton" name="page_email">
       <property name="geometry">
        <rect>
         <x>950</x>
         <y>390</y>
         <width>191</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>OCR A Extended</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(35, 170, 102);</string>
       </property>
       <property name="text">
        <string>Send an Email</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="search_line_edit">
       <property name="geometry">
        <rect>
         <x>580</x>
         <y>10</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 207);
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="destinataireEmail">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>380</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="objetEmail">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>420</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="bodyEmail">
       <property name="geometry">
        <rect>
         <x>720</x>
         <y>380</y>
         <width>181</width>
         <height>61</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(35, 40, 3, 255), stop:0.16 rgba(136, 106, 22, 255), stop:0.225 rgba(166, 140, 41, 255), stop:0.285 rgba(204, 181, 74, 255), stop:0.345 rgba(235, 219, 102, 255), stop:0.415 rgba(245, 236, 112, 255), stop:0.52 rgba(209, 190, 76, 255), stop:0.57 rgba(187, 156, 51, 255), stop:0.635 rgba(168, 142, 42, 255), stop:0.695 rgba(202, 174, 68, 255), stop:0.75 rgba(218, 202, 86, 255), stop:0.815 rgba(208, 187, 73, 255), stop:0.88 rgba(187, 156, 51, 255), stop:0.935 rgba(137, 108, 26, 255), stop:1 rgba(35, 40, 3, 255));
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>380</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>to:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>420</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>objet:</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1300</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
